srcStatic/GameModules/IniModule.cpp

```diff
@@ -19,12 +23,17 @@ IniModule::IniModule(IniSection iniSection)
 
 std::string IniModule::Directory()
 {
-	auto dllSetting = DllName();
-	return fs::path(dllSetting).remove_filename().string();
+	fs::path iniSetting(iniSection.GetValue("Path"));
+
+	if (iniSetting.empty()) {
+		iniSetting = DllName();
+		iniSetting.remove_filename();
+	}
+
+	return iniSetting.string();
 }
 
 std::string IniModule::DllName()
 {
-	auto defaultDllName = (fs::path(Name()) / "op2mod.dll").string();
-	return iniSection.GetValue("Dll", defaultDllName);
+	return iniSection.GetValue("Dll");
 }
```

Comment:
Not too sure why this part is being changed. I may be getting too tired to review though.

Reply:
This is to facilitate non-DLL mods, e.g. language mods. Such mods can either specify e.g. `Path = lang/it` under the mod's ini section, or as
```[ExternalModules]
lang/it = yes```

Reply:
I'm pretty sure we already had support for non-DLL mods. In particular, resource mods. Though perhaps support was asymmetric for the different mod loading methods. This may bear closer scrutiny.